src/elements/common/item/IconCell.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
let's write this file and the test in TS if possible

Reply:
I think it would be a larger refactor outside the scope of this PR, since it has dependencies on files that are not in typescript.

Reply:
Would it be easily resolved by using the comment Trevor told me about https://github.com/box/box-ui-elements/blob/004407dde579ac5d02f0c040ee0541c4b8189039/src/components/time-input/TimeInput.tsx#L7?

Reply:
Cool! The ts-ignore works for most of the issues.

But unfortunately the component also depends on a flow type `BoxItem`. That would need to be migrated to TS first.